node/cmd/guardiand/adminclient.go

```diff
@@ -137,6 +139,13 @@ var ClientChainGovernorResetReleaseTimerCmd = &cobra.Command{
 	Args:  cobra.ExactArgs(1),
 }
 
+var PurgePythNetVaasCmd = &cobra.Command{
+	Use:   "purge-pythnet-vaas [DAYS_OLD] <logonly>",
+	Short: "Deletes PythNet VAAs from the database that are more than [DAYS_OLD] days only (if logonly is specified, doesn't delete anything)",
+	Run:   runPurgePythNetVaas,
+	Args:  cobra.RangeArgs(1, 2),
```

Comment:
The answer to this is probably "no", but is there a way to specify that if there is only 1 arg, it must be the DAYS_OLD arg?